app/code/Magento/InventoryAdminUi/Test/Mftf/Test/LoggedInCustomerOrderGroupedProductCustomStock.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="LoggedInCustomerOrderGroupedProductCustomStock" extends="AdminCreateGroupedProductCustomStock">
```

Comment:
Is it optimal inheritance strategy? Scenarios seem absolutely different

Reply:
Will be refactored in next PR